math/mean.go

```diff
@@ -0,0 +1,25 @@
+package math
+
+import (
+	"github.com/TheAlgorithms/Go/constraints"
+)
+
+type Digit interface {
```

Comment:
This is useless if you're using the `constraints` package.

We already have `constraints.Number` which is exactly the same as this. Please use that instead and remote this.

Reply:
Ya, I forgot to remove them in my last push. Sorry for that.